src/sentry/replays/endpoints/organization_replay_details.py

```diff
@@ -15,11 +17,68 @@
 from sentry.apidocs.utils import inline_sentry_response_serializer
 from sentry.constants import ALL_ACCESS_PROJECTS
 from sentry.models.organization import Organization
+from sentry.replays.lib.eap import read as eap_read
+from sentry.replays.lib.eap.snuba_transpiler import RequestMeta, Settings
 from sentry.replays.post_process import ReplayDetailsResponse, process_raw_response
 from sentry.replays.query import query_replay_instance
 from sentry.replays.validators import ReplayValidator
 
 
+def query_replay_instance_eap(
+    project_ids: list[int],
+    replay_id: str,
+    start: datetime,
+    end: datetime,
+    organization_id: int | None = None,
+    referrer: str = "replays.query.details_query",
+):
+    snuba_query = Query(
+        match=Entity("replays"),
+        select=[
+            Column("replay_id"),
+            Column("project_id"),
+            Column("timestamp"),
+            Column("segment_id"),
+            Column("is_archived"),
```

Comment:
What is this value? Since we can't update data in EAP (at least not really), this will likely always be `false` since it'll be how it's inserted. You might not want it or need it.